### PR TITLE
made it so that alice can be run by gradlew on the command line

### PR DESCRIPTION
**Pull request by TrevorDecker**
also created a more efficient raceday robot

to run frome gralew go to alice and run ./gradlew run

### CODE REVIEW

**Comment by TrevorDecker** [comment]
@VivaanBahl can you check this

**Comment by VivaanBahl** [comment]
I am ok with this in terms of source changes, will pull and test soon

**Comment by VivaanBahl** [comment]
@TrevorDecker the gradlew additions seem to try to run Tutorial and Buggyros as well, is there a way to isolate this to Alice?

**Comment by TrevorDecker** [comment]
are you running from alice or from javasrc

**Comment by VivaanBahl** [comment]
from java_src

**Comment by TrevorDecker** [comment]
to run frome gralew go to alice and run ./gradlew run

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/RaceDayRobot.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import com.roboclub.robobuggy.nodes.sensors.HillCrestImuNode;
+import com.roboclub.robobuggy.nodes.sensors.LoggingNode;
+import com.roboclub.robobuggy.nodes.sensors.RBSMNode;
+import com.roboclub.robobuggy.ros.NodeChannel;
+import com.roboclub.robobuggy.ui.AutonomousPanel;
+import com.roboclub.robobuggy.ui.ConfigurationPanel;
+import com.roboclub.robobuggy.ui.Gui;
+import com.roboclub.robobuggy.ui.MainGuiWindow;
+import com.roboclub.robobuggy.ui.PathPanel;
+import com.roboclub.robobuggy.ui.PoseGraphsPanel;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/RaceDayRobot.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.ui.PoseGraphsPanel'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2376719565/issues/source?bid=2938665&fileBranchId=3224674#l21)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/RaceDayRobot.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+
+/**
+ * A robot class for having transistor drive itself
+ *
+ * @author Trevor Decker
+ */
+public final class RaceDayRobot extends AbstractRobot {
+	private static RaceDayRobot instance;
+	private static final int ARDUINO_BOOTLOADER_TIMEOUT = 2000;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/RaceDayRobot.java]: Codacy Issue found: [Avoid excessively long variable names like ARDUINO_BOOTLOADER_TIMEOUT](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2376719565/issues/source?bid=2938665&fileBranchId=3224674#l37)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/RaceDayRobot.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+import com.roboclub.robobuggy.nodes.localizers.HighTrustGPSLocalizer;
+import com.roboclub.robobuggy.nodes.planners.WayPointFollowerPlanner;
+import com.roboclub.robobuggy.nodes.planners.WayPointUtil;
+import com.roboclub.robobuggy.nodes.sensors.CameraNode;
+import com.roboclub.robobuggy.nodes.sensors.GpsNode;
+import com.roboclub.robobuggy.nodes.sensors.HillCrestImuNode;
+import com.roboclub.robobuggy.nodes.sensors.LoggingNode;
+import com.roboclub.robobuggy.nodes.sensors.RBSMNode;
+import com.roboclub.robobuggy.ros.NodeChannel;
+import com.roboclub.robobuggy.ui.AutonomousPanel;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/RaceDayRobot.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.ui.AutonomousPanel'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2376719565/issues/source?bid=2938665&fileBranchId=3224674#l16)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/RaceDayRobot.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+import com.roboclub.robobuggy.ui.ConfigurationPanel;
+import com.roboclub.robobuggy.ui.Gui;
+import com.roboclub.robobuggy.ui.MainGuiWindow;
+import com.roboclub.robobuggy.ui.PathPanel;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/RaceDayRobot.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.ui.PathPanel'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2376719565/issues/source?bid=2938665&fileBranchId=3224674#l20)

**Comment by VivaanBahl** [comment]
This one's gone stale, but we should definitely look into doing more with gradle in the future - this could really help get our launch latency down

**Pull request closed without merging**